### ISSUE TITLE
Thought Scour ability clarification and UI enhancement.

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#434]
When I play Thought Scour the ability is "Target player puts the top two cards of his or her library into his or her graveyard". If I was playing this in RL would I be allowed to look at those cards as I move them from the library pile to the graveyard? And is your opponent allowed to look at them? At the moment the cards are just moved into the graveyard and I have to manually select the graveyard zone to look at them. It would be much nicer if the UI was to briefly display the cards it is moving (similar to the animation for showing the card played by the AI as it does now).

**Comment by ShawnieBoy** [comment]
Yes, they would each be revealed as they entered the graveyard. Having an animation that matched the 'play a card' would be perfect. In a real-life way, each card revealed, then put into the graveyard sequentially (even though in game rules they all hit the graveyard at the same time)

**Comment by lodici** [comment]
Thanks for the clarification.
